composer.json

```diff
@@ -11,13 +11,13 @@
     ],
     "require": {
         "php": ">=7.2",
-        "laravel/framework": "^5.5||^6.0||^7.0||^8.0||^9.0",
+        "laravel/framework": "^5.5||^6.0||^7.0||^8.0||^9.0|^10.0",
         "divineomega/laravel-password-exposed-validation-rule": "^2.5.0"
```

Comment:
Update this to use laravel-shift:l10-compatibility.

Check DivineOmega Link -> https://github.com/DivineOmega/laravel-password-exposed-validation-rule/pull/27